gdnative-core/src/core_types/vector2.rs

```diff
@@ -21,6 +21,14 @@ pub trait Vector2Godot {
     fn tangent(self) -> Self;
     /// Returns `self` moved towards `to` by the distance `delta`, clamped by `to`.
     fn move_towards(self, to: Vector2, delta: f32) -> Self;
+    /// Returns the normalized vector pointing from this vector to `other`.
+    fn direction_to(self, other: Vector2) -> Vector2;
+    /// Returns the distance to `other`.
+    fn distance_to(self, other: Vector2) -> f32;
+    /// Returns the squared distance to `other`.
+    /// This method runs faster than distance_to, so prefer it if you need to compare vectors or
```

Comment:
An empty doc-comment line should be inserted between the summary and detailed descriptions.